### DISCUSSION TITLE
Progress bar inside a table

### DISCUSSION THREAD

**Discussion by nilmarpublio** [quasarframework/quasar#11334] [General - Components / Directives / etc] [answered]
Hello everyone,
I'm learning to use quasar with vue and I need help with the following:
1- I have a list of files to process.
2- I want that when selecting each file, place a progressbar in a table to indicate how much of each file has been processed.

Can someone help me ?

**Comment by metalsadman** [comment]
show a codepen how it initially looks. https://quasar.dev/vue-components/table#loading-state
or use slots along with https://quasar.dev/vue-components/linear-progress

**Comment by nilmarpublio** [comment]
Thanks,
I kew that, my doubt is how do I put the Progress inside a table row.



Em dom., 14 de nov. de 2021 às 22:29, Aldrin Marquez <
***@***.***> escreveu:

> show a codepen how it initially looks.
> https://quasar.dev/vue-components/table#loading-state
> or use slots along with https://quasar.dev/vue-components/linear-progress
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quasarframework/quasar/discussions/11334#discussioncomment-1641900>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEXHXA6WIAKUU3SVDBMAODUMBO7DANCNFSM5H77RRPA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Nilmar P. Castro
Cel: (61) 99555-2460

**Comment by metalsadman** [comment, marked as answer]
https://codepen.io/metalsadman/pen/PoKXmGN?editors=1010

**Comment by nilmarpublio** [comment]
Ok Aldrin. Thanks so much.


Em ter., 16 de nov. de 2021 às 22:45, Aldrin Marquez <
***@***.***> escreveu:

> https://codepen.io/metalsadman/pen/PoKXmGN?editors=1010
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quasarframework/quasar/discussions/11334#discussioncomment-1655127>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEXHXFQWOCJ7UDFXUECQL3UMMCNFANCNFSM5H77RRPA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Nilmar P. Castro
Cel: (61) 99555-2460
